src/components/nav-bar/nav-bar.js

```diff
@@ -38,6 +49,26 @@ const NavBar = () => {
     dispatch(setSideMenuStatus(!sideMenuStatus));
   };
 
+  const openSuccessSnackbar = (onClickHandler) => {
```

Comment:
It is a pretty useless function because `openSuccessSnackbar` is used only once. Please, inline this function in a place where it is used

Reply:
This function will be used in many places, so I created custom hook called useSuccessSnackBar, that contains openSuccessSnackbar function. It's good?